cluster/gce/config-test.sh

```diff
@@ -367,6 +367,9 @@ ENABLE_PROMETHEUS_TO_SD="${ENABLE_PROMETHEUS_TO_SD:-true}"
 # Optional: Run kube-proxy as a DaemonSet if set to true, run as static pods otherwise.
 KUBE_PROXY_DAEMONSET="${KUBE_PROXY_DAEMONSET:-false}" # true, false
 
+# Optional: duration of cluster signed certificates.
+CLUSTER_SIGNING_DURATION="${CLUSTER_SIGNING_DURATION:-30m}"
```

Comment:
This should not be enabled for all test clusters by default.

Reply:
I'm curious, why not?

Reply:
IIUC (which may not be the case), kubelet needs to rotate client certificate and it's not enabled by default.

Reply:
So why do we then revert vs actually enabling kubelet cert rotation on kubelets?
cc @ericchiang 
(Not opposed, just asking)

Reply:
@luxas I really just want to get the tests back to green as soon as possible. Once that's done happy to see other PRs modifying the test setup.

Reply:
SGTM
@jcbsmpsn probably can create an issue to follow this up later

Reply:
Yes, reverting is the standard practice in cases like this.

Also, I am unsure whether this should be tested in a separate job, or enabled by default for all test clusters.